packages/input/src/Input.js

```diff
@@ -22,23 +22,18 @@ export const SuffixContainer = styled.span`
 `
 
 export const SpinnerContainer = SuffixContainer.extend`
-  top: 12px;
+  top: ${({ mini }) => (mini ? '5px' : '12px')};
```

Comment:
You like 5px instead of 6px?

Reply:
It was a toss up between them, if I’m honest. If you like 6px, that’s good too :)

Reply:
Wasn't sure with your new linux browser experience 😉 It is a bit on the odd side either way.

Reply:
:laughing: